modules/overview/pages/tokens.adoc

```diff
@@ -0,0 +1,164 @@
+
+There are two types of Skupper token:
+
+Claim token (default):: 
++
+--
+A claim token can be restricted by:
+
+* time - prevents token reuse after a specified period.
+* usage - prevents creating multiple links from a single token.
+
+All inter-site traffic is protected by mutual TLS using a private, dedicated certificate authority (CA).
+A claim token is not a certificate, but is securely exchanged for a certificate during the linking process.
+By implementing appropriate restrictions (for example, creating a single-use claim token), you can avoid the accidental exposure of certificates.
+--
+
+Cert token ::
++
+--
+You can use a cert token to create a link to the site which issued that token, it includes a valid certificate from that site.
+
+All inter-site traffic is protected by mutual TLS using a private, dedicated certificate authority (CA).
+A cert token is a certificate issued by the dedicated CA.
+Protect it appropriately.
+--
+
+// Type: procedure
+[id='creating-claim-tokens-{context}']
+== Creating claim tokens
+
+[role="system:abstract"]
+You can use a claim token to create a link to the site which issued that token.
+It does not includes a certificate from that site, but a certificate is passed from the site when the claim token is used.
+A claim token can be restricted by time or usage.
+
+.Procedure
+
+. Log into the cluster.
+
+. Change to the namespace associated with the site.
+
+. Create a claim token, for example:
++
+--
+[source,bash,options="nowrap"]
+----
+$ skupper token create $HOME/secret.yaml --expiry 30m0s --uses 2 -t claim
+----
+
+NOTE: Claim tokens are the default, the `-t claim` section of the command is unnecessary.
+
+--expiry:: The amount of time the token is valid in minutes and seconds, default `15m0s`.
+--uses:: The number of times you can use the token to create a  link, default `1`.
+--
+
+.Additional information
+
+* See the {CliLink} for information on using the token to create links.
+
+// Type: procedure
+[id='creating-claim-tokens-{context}']
+== Creating cert tokens
+
+[role="system:abstract"]
+A cert token allows you create many links to the {service-network} from different sites without restrictions.
+
+.Procedure
+
+. Log into the cluster.
+
+. Change to the namespace associated with the site.
+
+. Create a cert token:
++
+--
+[source,bash,options="nowrap"]
+----
+$ skupper token create $HOME/secret.yaml -t cert
+----
+
+NOTE: Cert tokens are always valid and can be reused indefinitely unless revoked as described in xref:revoking-access-{context}[]
+--
+
+.Additional information
+
+* See the {CliLink} for information on using the token to create links.
+
+// Type: procedure
+[id='revoking-access-{context}']
+== Revoking access to a site
+
+[role="system:abstract"]
+To secure access to the {service-network} you can invalidate all the tokens created from a site.
```

Comment:
Rtaher than 'to secure access' I would say, something like 'should a token by compromised, you can prevent unauthorised use of that token by invalidating all tokens'